spec/services/google/keyword_service_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::KeywordService, type: :service do
+  describe '#call' do
+    context 'given a keyword with valid result' do
+      it 'returns 3 top ads count' do
```

Comment:
```suggestion
      it 'returns 3 as top ads count' do
```

or `it has a top ads count of 3`. 
Otherwise one might understand there are 3 different values returned as top ads count.